web/fanfare/src/core/manager.ts

```diff
@@ -2,24 +2,29 @@ import { Observable } from "rxjs";
 import { Transaction } from "../types/OutputMessage";
 import { TransactionProvider } from "../types/Transaction";
 import { onUnhealthy } from "./unhealthy";
-
-import config from "../config";
+import { FanfareConfig } from "../types/Config";
 
 export const manager = (
     provider: TransactionProvider,
+    { 
+        debug,
+        unhealthyThreshold
+    }: FanfareConfig,
 ) => {
     let failureCount = 0;
 
+    debug && console.log(`[debug] Starting manager for ${provider.name()}...`);
```

Comment:
TODO: add a debugging facility to our internal logging...

Reply:
(For the frontend at least)

Reply:
It's kind of an injection problem with this one

Reply:
Need to find a way to pass down config without a dependency cycle